src/Injector/inject.cpp

```diff
@@ -0,0 +1,41 @@
+#include <iostream>
+#include <Windows.h>
+
+int inject(DWORD PID)
+{
+    // path to dll
+    //LPCSTR DllPath = "../../Hooks/debug/Hooks.dll";
+    LPCSTR DllPath = "C:\\CodeBase\\Apriorit-Project\\build-src-Desktop_Qt_6_3_1_MinGW_64_bit-Debug\\Hooks\\debug\\Hooks.dll";
+
+
+    // Open a handle to target process
+    HANDLE hProcess = OpenProcess(PROCESS_ALL_ACCESS, FALSE, PID);
```

Comment:
Check the result of this and next functions

Reply:
done